src/modules/tests/tests.reducer.ts

```diff
@@ -0,0 +1,49 @@
+import * as preTestDeclarationActions from './pre-test-declarations/pre-test-declarations.actions';
+import * as testOutcomeActions from '../../pages/journal/components/test-outcome/test-outcome.actions';
+import { preTestDeclarationsReducer } from './pre-test-declarations/pre-test-declarations.reducer';
+import { candidateReducer } from './candidate/candidate.reducer';
+import { get } from 'lodash';
+import { combineReducers } from '@ngrx/store';
+
+// Extend this with any new test domain action types
+type TestAction = preTestDeclarationActions.Types | testOutcomeActions.Types;
+
+const initialState = {};
```

Comment:
Can we add some type safety to this?

Reply:
Ah yes, I knew there was another thing I wanted to do! 😁

Reply:
Type safety added.